content/rest-api/filter.md

```diff
@@ -1253,6 +1253,15 @@ Note that dates should follow the [ISO 8601 standard](https://en.wikipedia.org/w
 | -------- | -------------------------------- | ------------------------------------------------------------------------------------ |
 | `>`      | datetime <br> _Format: ISO 8601_ | Only returns records that were respectively<br> updated after the given day and hour |
 
+::: availability versions=6.0,SaaS editions=EE
```

Comment:
```suggestion
::: availability versions=SaaS editions=EE
```

your change is targeting master so it won't be available for 6.0 tagged version.